src/extension.ts

```diff
@@ -98,20 +98,22 @@ export async function activate(context: vscode.ExtensionContext) {
 				if (changeEvent.changed.length > 0) {
 					const oldPath = CodeSyncState.get(CODESYNC_STATES.ACTIVE_TAB_PATH);
 					// @ts-ignore
+					if  (!changeEvent.changed[0]?.input || !changeEvent.changed[0]?.input?.uri || !changeEvent.changed[0]?.input?.uri.path) return 
```

Comment:
If this is becoming a longer line, we should shorten it by defining a variable or maybe using multiple lines